documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1732,6 +1755,22 @@ Without these grants, the connector cannot operate.
 
 |===
 
+=== Standby databases
+
+An Oracle database can be configured with either a physical or a logical standby environment to provide for recovery after of a production failure.
+At this time, the {prodname} Oracle connector cannot use a physical or logical standby database as the change event source.
+There is an open https://issues.redhat.com/browse/DBZ-3866[Jira issue] to investigate this support.
+
+=== Failover databases
+
+It is customary for a logical or physical standby to exist in the case of an Oracle production failure.
+When a failure occurs and the standby instance is promoted to production, the database must be opened for read/write transactions before the {prodname} Oracle connector can connect to the database.
+
+In the case of a physical standby, the standby is an exact copy of production, which implies that the SCN values are identical.
+When using a physical standby, it is sufficient to reconfigure the {prodname} Oracle connector to use the hostname of the standby once the database is open.
+
+In the case of a logical standby, the standby is not an exact copy of the production database, so the SCN offsets in the standby differ from those in the production database.
+If you use a logical standby, to guarantee that no change events are lost, after the database is open, configure a new connector and perform a new database snapshot.
```

Comment:
```suggestion
If you use a logical standby, to help ensure that {prodname} does not miss any change events, after the database is open, configure a new connector and perform a new database snapshot.
```
This snuck past me in my earlier review. From a legal perspective, we must be careful about using the word "guarantee", because of its specific it carries implications in the event that something goes awry. I hate to use weasel words, but unless we can offer users an explicit 100% assurance that something will work, it's best not to frame any behaviors as being guaranteed.

Reply:
That makes perfect sense and I'll be mindful of those words and their implied meanings moving forward.